docs/LICENSE.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: LICENSE
+nav_order: 9
+---
+
+MIT License
+
+Copyright (c) 2022 just-the-docs
```

Comment:
I think the LICENSE here is confusing, since it's the Just the Docs license, not the Metafacture license (which is Apache 2). I think this can be removed here, or replaced with (a link to) the Metafacture license: https://github.com/metafacture/metafacture-core/blob/master/LICENSE